tools/radius_filter.cpp

```diff
@@ -57,8 +57,8 @@ printHelp (int, char **argv)
 {
   pcl::console::print_error ("Syntax is: %s input.pcd output.pcd <options>\n", argv[0]);
   pcl::console::print_info ("  where options are:\n");
-  pcl::console::print_info ("                     -radius X = Radius of the spere to filter (default: ");
-  pcl::console::print_value ("%s", default_radius); pcl::console::print_info (")\n");
+  pcl::console::print_info ("                     -radius X = Radius of the sphere to filter (default: ");
+  pcl::console::print_value ("%d", default_radius); pcl::console::print_info (")\n");
```

Comment:
`default_radius` is a float, so `%d` is also wrong (`%d` is for signed decimal integers). For float and double, `%f` or `%g` would be appropriate. https://cplusplus.com/reference/cstdio/printf/

Reply:
god dammit